runtime/metrics/src/std.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Runtime metrics are wasm only so we also need to provide a dummy metric
+//! implementation for the native runtime. These are required to avoid cluttering
+//! the code with `#[cfg(not(feature = "std"))]` for updating metrics.
+pub struct Counter;
```

Comment:
```suggestion

pub struct Counter;
```

Reply:
Docs are missing here and for the other types, also in nostd.

Reply:
fixed